shadows/framework/src/main/java/org/robolectric/shadows/ShadowTelephonyManager.java

```diff
@@ -701,7 +708,7 @@ protected void initListener(PhoneStateListener listener, int flags) {
   }
 
   @CallSuper
-  protected void initTelephonyCallback(TelephonyCallback callback) {
+  protected void initTelephonyCallback(Object callback) {
```

Comment:
Oh, it might also needs a type checking. I think it is the last needed changes from my perspective.

Reply:
initTelephonyCallback is protected and only called by registerTelephonyCallback, I think type checking is not necessary in this method

Reply:
Okay, it's fine.